app/src/main/res/values/strings.xml

```diff
@@ -159,6 +159,7 @@
     <string name="dialog_create_account_checking_progress">Verifying</string>
     <string name="create_account_email_hint">Email (Optional)</string>
     <string name="create_account_password_repeat_hint">Repeat password</string>
+    <string name="create_account_password_is_username">Password can\'t be the same as username</string>
```

Comment:
When adding a string resource to `strings.xml`, you must also add the same resource to `values-qq\strings.xml`, with an explanation of the string.

This is part of our community translation architecture ([translatewiki.net](https://translatewiki.net)) that helps community members translate strings into all the languages we support.

Reply:
Thanks for pointing that out, the string is added now